feature-libs/asm/styles/components/_customer-selection.component.scss

```diff
@@ -13,6 +13,12 @@
       margin: 0 0 15px;
       min-width: auto;
 
+      @include forVersion(6.1) {
+        cx-form-errors p {
+          color: #db0002;
```

Comment:
styling on tag level is not a good practice unless you want all target tags to apply this style.

think about below question:
1. is there only one `p` tag under `cx-form-errors`?
2. what if we introduce another `p` tag under `cx-form-errors`, current color will apply to that tag also, is it expected?

Reply:
The reason for the color change is that ASM needs to meet the Horizon style, and the text that changes color is wrapped by the `cx-form-errors` public component. 
The `cx-form-errors` is a public component and it is not possible to add classes to it freely. Therefore, the current quickest solution may be to use hard-coding to solve the problem.

Reply:
got it, that make sense :thumbup